aten/src/TH/THGeneral.h.in

```diff
@@ -80,7 +85,13 @@ typedef struct {
 TH_API double THLog1p(const double x);
 TH_API double THLog2(const double x);
 TH_API double THExpm1(const double x);
-TH_API THDescBuff _THSizeDesc(const int64_t *size, const int64_t ndim);
+
+#ifdef __cplusplus
+// Mangled so we can have an overload
+AT_API THDescBuff _THSizeDesc(const int64_t *size, const int64_t ndim);
+AT_API THDescBuff _THSizeDesc(at::ArrayRef<int64_t> size, const int64_t ndim);
```

Comment:
just give these different names?

Reply:
OK, thirty call sites is not too bad, I'll just fix them.